website/docs/r/virtual_machine_scale_set.html.markdown

```diff
@@ -260,6 +260,7 @@ The following arguments are supported:
 * `extension` - (Optional) Can be specified multiple times to add extension profiles to the scale set. Each `extension` block supports the fields documented below.
 * `boot_diagnostics` - (Optional) A boot diagnostics profile block as referenced below.
 * `plan` - (Optional) A plan block as documented below.
+* `priority` - (Optional) Specifies the priority for the virtual machines in the scale set. Possible values are `Low` and `Regular` with the default being `Regular`.
```

Comment:
" with the default being `Regular`". ->  ". Defaults to `Regular`" to match the other resources